centreon/package.json

```diff
@@ -133,7 +133,7 @@
     "ramda": "^0.28.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-i18next": "^12.0.0",
+    "react-i18next": "11.15.1",
```

Comment:
Why downgrading to 11?

Reply:
for some reason t(labelOfSomething) is not a string, so we have to put something like that each time t(labelOfSomething) as string. in centreon-modules we have the version 11

Reply:
You can add as string then